vana/wallet.py

```diff
@@ -227,6 +227,65 @@ def __init__(
         self._coldkeypub = None
         self._mnemonics = {}
 
+        # Create necessary directories
+        self._create_wallet_directories()
+
+        # Check for and handle environment variable based wallet initialization
+        self._init_from_environment()
+
+    def _create_wallet_directories(self):
```

Comment:
I think some of this setup is already done? 

https://github.com/vana-com/vana-framework/blob/main/vana/wallet.py#L277